module_3/gift_certificates_system/model/src/main/java/com/epam/esm/entity/MostWidelyUsedTag.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.esm.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+/**
+ * Contain most widely used tag for user with the highest cost of all orders.<br>
+ * Also contain set of tag and the number of times these tags have been used by the user.
+ * @see com.epam.esm.entity.UserPrice
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class MostWidelyUsedTag {
```

Comment:
It's not an entity, why is it in the "entity" package?

Reply:
Thanks, removed from `entity` to `pojo` package.